### PR TITLE
gui: Fix allocations dialog typo and incorrect type

### PR DESCRIPTION
**Pull request by EXtremeExploit**
ranges are 32 bit, not 64, also segment indicator wasnt working
image

### CODE REVIEW

File: vita3k/gui/src/allocations_dialog.cpp [hunk 1]
```diff
@@ -48,7 +48,7 @@ void draw_allocations_dialog(GuiState &gui, EmuEnvState &emuenv) {
             continue;
 
         if (ImGui::TreeNode(fmt::format("{}: {}", generation_num, generation_name).c_str())) {
-            ImGui::Text("Range %08llx - %08llx.", generation_num * KiB(4), (generation_num + page.size) * KiB(4));
```
Comment (Macdu) [inline vita3k/gui/src/allocations_dialog.cpp:51]: This is wrong, generation_num * KiB(4) is a size_t, so the quantifier should be llx (or zx), lx works on Linux but not on Windows.

**Comment by Macdu** [comment]
What is the issue? Without this commit it already displays correctly (for the range part):
image

**Comment by EXtremeExploit** [comment]
Mainly was to fix a warning:
```
[build] [1/5] Building CXX object vita3k/gui/CMakeFiles/gui.dir/Debug/src/allocations_dialog.cpp.o
[build] /home/pedro/dev/Vita3K/vita3k/gui/src/allocations_dialog.cpp:51:51: warning: format specifies type 'unsigned long long' but the argument has type 'unsigned long' [-Wformat]
[build]             ImGui::Text("Range %08llx - %08llx.", generation_num * KiB(4), (generation_num + page.size) * KiB(4));
[build]                                ~~~~~~             ^~~~~~~~~~~~~~~~~~~~~~~
[build]                                %08lx
[build] /home/pedro/dev/Vita3K/vita3k/gui/src/allocations_dialog.cpp:51:76: warning: format specifies type 'unsigned long long' but the argument has type 'unsigned long' [-Wformat]
[build]             ImGui::Text("Range %08llx - %08llx.", generation_num * KiB(4), (generation_num + page.size) * KiB(4));
[build]                                         ~~~~~~                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[build]                                         %08lx
[build] 2 warnings generated.
```
but i also noticed that the end of every allocation (the elf segment) wasn't working

File: vita3k/gui/src/allocations_dialog.cpp (hunk 1 above)
Comment (EXtremeExploit) [reply vita3k/gui/src/allocations_dialog.cpp:51]: is this enough? 
```cpp
ImGui::Text("Range 0x%08zx - 0x%08zx.", generation_num * KiB(4), (generation_num + page.size) * KiB(4));
```

Comment (Macdu) [reply vita3k/gui/src/allocations_dialog.cpp:51]: Yes, this should work. Or static cast them back to uint32_t and use %08x, as you want

**Comment by EXtremeExploit** [comment]
this should be good now <:

**Review by Macdu — APPROVED** [review]

**Pull request merged**